### ISSUE TITLE
Cannot build libxavs

### ISSUE THREAD

**Issue by uberware** [jb-alvarado/media-autobuild_suite#125]
Using this system to build I get through everything until I get to xavs, which fails to build and give me the following messages:

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   129    0   129    0     0    258      0 --:--:-- --:--:-- --:--:--   275
100  316k  100  316k    0     0   269k      0  0:00:01  0:00:01 --:--:--  269k
Archive:  distrotech-xavs.zip
8300fbf4918bca2af1af2aee6a337815e90b51e1
   creating: xavs-distrotech-xavs/
  inflating: xavs-distrotech-xavs/Makefile
   creating: xavs-distrotech-xavs/build/
   creating: xavs-distrotech-xavs/build/win32/
  inflating: xavs-distrotech-xavs/build/win32/libxavs.sln
  inflating: xavs-distrotech-xavs/build/win32/libxavs.vcproj
  inflating: xavs-distrotech-xavs/build/win32/xavs.sln
  inflating: xavs-distrotech-xavs/build/win32/xavs.vcproj
   creating: xavs-distrotech-xavs/common/
  inflating: xavs-distrotech-xavs/common/bs.h
  inflating: xavs-distrotech-xavs/common/common.c
  inflating: xavs-distrotech-xavs/common/common.h
  inflating: xavs-distrotech-xavs/common/cpu.c
  inflating: xavs-distrotech-xavs/common/cpu.h
  inflating: xavs-distrotech-xavs/common/csp.c
  inflating: xavs-distrotech-xavs/common/csp.h
  inflating: xavs-distrotech-xavs/common/dct.c
  inflating: xavs-distrotech-xavs/common/dct.h
  inflating: xavs-distrotech-xavs/common/frame.c
  inflating: xavs-distrotech-xavs/common/frame.h
   creating: xavs-distrotech-xavs/common/i386/
  inflating: xavs-distrotech-xavs/common/i386/cpu-a.asm
  inflating: xavs-distrotech-xavs/common/i386/dct-c.c
  inflating: xavs-distrotech-xavs/common/i386/dct-sse2.asm
  inflating: xavs-distrotech-xavs/common/i386/dct.c
  inflating: xavs-distrotech-xavs/common/i386/dct.h
  inflating: xavs-distrotech-xavs/common/i386/deblock.asm
  inflating: xavs-distrotech-xavs/common/i386/deblock_inter.asm
  inflating: xavs-distrotech-xavs/common/i386/deblock_intra.asm
  inflating: xavs-distrotech-xavs/common/i386/i386inc.asm
  inflating: xavs-distrotech-xavs/common/i386/mc-a.asm
  inflating: xavs-distrotech-xavs/common/i386/mc-a01.asm
  inflating: xavs-distrotech-xavs/common/i386/mc-c.c
  inflating: xavs-distrotech-xavs/common/i386/pixel-a.asm
  inflating: xavs-distrotech-xavs/common/i386/pixel-sse2.asm
  inflating: xavs-distrotech-xavs/common/i386/pixel.h
  inflating: xavs-distrotech-xavs/common/i386/predict-a.asm
  inflating: xavs-distrotech-xavs/common/i386/predict-c.c
  inflating: xavs-distrotech-xavs/common/i386/predict.h
  inflating: xavs-distrotech-xavs/common/i386/qpel.asm
  inflating: xavs-distrotech-xavs/common/i386/quant_sse2.asm
  inflating: xavs-distrotech-xavs/common/macroblock.c
  inflating: xavs-distrotech-xavs/common/macroblock.h
  inflating: xavs-distrotech-xavs/common/mc.c
  inflating: xavs-distrotech-xavs/common/mc.h
  inflating: xavs-distrotech-xavs/common/mdate.c
  inflating: xavs-distrotech-xavs/common/osdep.h
  inflating: xavs-distrotech-xavs/common/pixel.c
  inflating: xavs-distrotech-xavs/common/pixel.h
  inflating: xavs-distrotech-xavs/common/predict.c
  inflating: xavs-distrotech-xavs/common/predict.h
  inflating: xavs-distrotech-xavs/common/quant.c
  inflating: xavs-distrotech-xavs/common/quant.h
  inflating: xavs-distrotech-xavs/common/set.c
  inflating: xavs-distrotech-xavs/common/set.h
  inflating: xavs-distrotech-xavs/common/vlc.h
  inflating: xavs-distrotech-xavs/config.guess
  inflating: xavs-distrotech-xavs/config.sub
  inflating: xavs-distrotech-xavs/configure
   creating: xavs-distrotech-xavs/doc/
 extracting: xavs-distrotech-xavs/doc/ratecontrol.txt
  inflating: xavs-distrotech-xavs/doc/regression_test.txt
 extracting: xavs-distrotech-xavs/doc/standards.txt
 extracting: xavs-distrotech-xavs/doc/threads.txt
 extracting: xavs-distrotech-xavs/doc/vui.txt
   creating: xavs-distrotech-xavs/encoder/
  inflating: xavs-distrotech-xavs/encoder/analyse.c
  inflating: xavs-distrotech-xavs/encoder/analyse.h
  inflating: xavs-distrotech-xavs/encoder/cavlc.c
  inflating: xavs-distrotech-xavs/encoder/encoder.c
  inflating: xavs-distrotech-xavs/encoder/eval.c
  inflating: xavs-distrotech-xavs/encoder/lookahead.c
  inflating: xavs-distrotech-xavs/encoder/macroblock.c
  inflating: xavs-distrotech-xavs/encoder/macroblock.h
  inflating: xavs-distrotech-xavs/encoder/me.c
  inflating: xavs-distrotech-xavs/encoder/me.h
  inflating: xavs-distrotech-xavs/encoder/ratecontrol.c
  inflating: xavs-distrotech-xavs/encoder/ratecontrol.h
  inflating: xavs-distrotech-xavs/encoder/rdo.c
  inflating: xavs-distrotech-xavs/encoder/set.c
  inflating: xavs-distrotech-xavs/encoder/set.h
  inflating: xavs-distrotech-xavs/encoder/slicetype.c
   creating: xavs-distrotech-xavs/extras/
  inflating: xavs-distrotech-xavs/extras/getopt.c
  inflating: xavs-distrotech-xavs/extras/getopt.h
  inflating: xavs-distrotech-xavs/extras/stdint.h
  inflating: xavs-distrotech-xavs/matroska.c
  inflating: xavs-distrotech-xavs/matroska.h
  inflating: xavs-distrotech-xavs/muxers.c
  inflating: xavs-distrotech-xavs/muxers.h
   creating: xavs-distrotech-xavs/tools/
  inflating: xavs-distrotech-xavs/tools/checkasm-a.asm
  inflating: xavs-distrotech-xavs/tools/checkasm.c
  inflating: xavs-distrotech-xavs/tools/countquant_xavs.pl
  inflating: xavs-distrotech-xavs/tools/patcheck
  inflating: xavs-distrotech-xavs/tools/q_matrix_avs.cfg
  inflating: xavs-distrotech-xavs/tools/regression-test.pl
  inflating: xavs-distrotech-xavs/tools/xavs-format
  inflating: xavs-distrotech-xavs/tools/xyuv.c
  inflating: xavs-distrotech-xavs/version.sh
  inflating: xavs-distrotech-xavs/xavs.c
  inflating: xavs-distrotech-xavs/xavs.h
Unknown option --build=x86_64-w64-mingw32, ignored
Unknown option --disable-shared, ignored
./configure: line 30: NUL: Permission denied
No working C compiler found.
Makefile:3: config.mak: No such file or directory
./configure
./configure: line 30: NUL: Permission denied
No working C compiler found.
Makefile:91: recipe for target 'config.mak' failed
make: *** [config.mak] Error 1
install: cannot stat ‘libxavs.a’: No such file or directory
install: cannot stat ‘xavs.pc’: No such file or directory
-------------------------------------------------
Build of xavs failed...
Delete the source folder under '/build' and start again.
```

I have tried removing it from the build folder and re-running the build process several times. I can live without this library, but would prefer to have it if possible. Does anyone have an idea why this is not working or what I can do to resolve the problem? 

Thank you,
-robin

**Comment by uberware** [comment]
Strange, but git checkout of latest version for me still had the issue. I manually corrected the configure script myself to resolve, but perhaps something got committed incorrectly?

**Comment by wiiaboo** [comment]
Could you rephrase that? What exactly did you correct?

**Comment by uberware** [comment]
On lines 202 and 211, I changed "NUL" to "/dev/null" and it built with no errors.

**Comment by wiiaboo** [comment]
Hm, makes sense. Don't know why it worked in my case.
